### PR TITLE
ADEN-2812 IVW3 tracker integration

### PR DESCRIPTION
**Pull request by fraszczakszymon**
Implementation of new IVW3 tracking provider which will be enabled in DE, AT, CH

### CODE REVIEW

File: front/common/modules/Trackers/IVW3.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import BaseTracker from './BaseTracker';
+
+/**
+ * @class Nielsen
```
Comment (rogatty) [inline front/common/modules/Trackers/IVW3.js]: You sure? ;)

Comment (fraszczakszymon) [inline front/common/modules/Trackers/IVW3.js]: :see_no_evil:

**Comment by jacekjursza** [comment]
LGTM

**Comment by jacekjursza** [comment]
LGTM

**Comment by jacekjursza** [comment]
:+1:

**Pull request merged**